app/models/purchase.rb

```diff
@@ -0,0 +1,5 @@
+class Purchase < ApplicationRecord
+  belongs_to :location
```

Comment:
If you would like to add receipt_file_upload capabilities to this model, just include the following:
```suggestion
  has_many_attached :receipt_file_uploads
  belongs_to :location
```

I think a Purchase can have many receipts? That's not clear to me. If it's just one, then you can do:
`has_one_attached :receipt_file_upload`

Reply:
will take care of this in:
https://github.com/rubyforgood/voices-of-consent/issues/75